mergin/client.py

```diff
@@ -137,14 +137,14 @@ def setup_logging(self):
                 log_handler.setFormatter(logging.Formatter('%(asctime)s %(message)s'))
                 self.log.addHandler(log_handler)
             else:
-                # no Mergin log path in the environment - create a null handler that does nothing
+                # no Mergin Maps log path in the environment - create a null handler that does nothing
                 null_logger = logging.NullHandler()
                 self.log.addHandler(null_logger)
 
     @staticmethod
     def default_url():
-        """ Returns URL of the public instance of Mergin """
-        return "https://public.cloudmergin.com"
+        """ Returns URL of the public instance of Mergin Maps"""
+        return "https://app.merginmaps.com/"
```

Comment:
you have added "/" at the end, I would not do that if it was working without "/" before?